### ISSUE TITLE
Selected text is wrongly deleted when the "Windows" key is pressed

### ISSUE THREAD

**Issue by vladikoff** [ckeditor/ckeditor5#955]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

CKEditor 5, v1.0.0-beta.2

## 📋 Steps to reproduce

1. Open https://ckeditor5.github.io/
2.Select the text and press the "Windows" key.
3. Observe the behavior.

## ✅ Expected result

The "Start" (Windows) or the "Search" (Linux) menu is opened.

## ❎ Actual result

The selected content is deleted and the "Start" (Windows) or the "Search" (Linux) menu is opened..

## 📃 Other details that might be useful

![](https://user-images.githubusercontent.com/31243259/38555402-99bb08a8-3cce-11e8-986b-652c97dbc869.gif)

Ref: https://github.com/mozilla/notes/issues/924

**Comment by pomek** [comment]
It could be related to https://github.com/ckeditor/ckeditor5-typing/issues/136.

**Comment by Reinmar** [comment]
Yep, seems to be a DUP of it. Closing, but please remember about these keys too when resolving ckeditor/ckeditor5-typing#136.

**Comment by vladikoff** [comment]
👍
